app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+from __future__ import annotations
+from .cafe import Cafe
+from .errors import (VaccineError,
+                     NotWearingMaskError)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> None | str:
+    masks_to_buy = sum(1 for friend in friends
+                       if not friend.get("wearing_a_mask"))
```

Comment:
it's a redundant calculation, which takes extra time, use variable and  +=1 every time when NotWearingMaskError handles

Reply:
Done